dmorg.tex

```diff
@@ -0,0 +1,97 @@
+
+Configuration control of documents is dealt with in \secref{sect:docman}. Here we consider more the operational systems and software configuration control.
+
+\subsubsection{Software Configuration Control}
+
+{\bf We should have a configuration management plan covering this.} \\
+
+DM follows a git based versioning system based  on public git repositories and the approach is covered in the developer guide \url{https://developer.lsst.io/}.
+The master branch is the stable code with development done in {\em ticket} branches (named with the id of the corresponding Jira Ticket describing the work.
+Once reviewed a branch is merged to master.\footnote{LSE-14 seem out of date and should be updated or revoked - titled a guideline it seems inappropriate as an LSE.}
+
+As we approach commissioning and operations DM will have a much stricter configuration control.
+At this point there will be a version of the software which may need urgent patching, a next candidate release version of the software, and the master.
+A patch to the operational version will require the same fix to be made in the two other versions.
+The role of the DM Change Control Board (DMCCB; \secref{sect:tct}) becomes very important at this point to ensure only essential fixes make it to the live system as patches and that required features are included in planned releases.
+
+We cannot escape the fact that we  will have multiple code branches to maintain in operations which will lead to an increase in work load.
+Hence one should consider that perhaps more manpower may be needed in commissioning to cope with urgent software fixes while continuing development.
+The other consideration would be that features to be developed post commissioning will probably be delayed more than one may think, as maintenance will take priority.\footnote{WOM identifies this as the maintenance surge.}
+
+\subsubsection{Hardware Configuration Control}
+
+On the hardware side we have multiple configurable items, we need to control which versions of software are on which machines. These days tooling like Puppet make this reasonably painless. Still the configuration  must be carefully controlled to ensure reproducible deployments providing correct and reproducible results. The exact set of released software and other tools on each system should be held in a configuration item list.
+Changes to the configuration should be endorsed by the DMCCB.
+
+\subsection {Risk Management } \label{sect:risk}
+
+Risks will be dealt with within the LSST Project framework as defined in \citeds{LPM-20}.
+Specific DM processes for risk assessment are elaborated in \citeds{LDM-512}.
+
+\subsection {Quality Assurance  } \label{sect:pa}
+
+In accordance with the project QA plan \citeds{LPM-55} we will perform QA on the software products.
+This work will mainly be carried out by SQuaRE (\secref{sect:square}).
+
+
+\subsection{Action item control}
+Actions in DM are tracked as Jira issues an periodically reviewed at DMLT meetings.
```

Comment:
We should probably state that these are action items arising from meetings. Are we planning to immediately write JIRA tickets for actions and missing the Confluence tasks intermediary? Are we using special labels for actions? Are the JIRA tickets linked to the confluence page for the meeting?